PythonAPI/examples/autoware_demo.py

```diff
@@ -506,21 +532,27 @@ def main():
         default=100,
         help="Set 'None' or 0 for variable time step, otherwise use an integer for fixed time step rate."
     )
+    argparser.add_argument(
+        '--list_maps', action='store_true',
+        help='Lists only available maps and exit. Omit applying world setting and ego spawn.')
     args = argparser.parse_args()
 
     # Get Client info
     client = carla.Client(args.host, args.port)
     client.set_timeout(60.0)
-    world = client.get_world()
+
+    if args.list_maps:
+        print("Available maps")
+        print(client.get_available_maps())
+        return
 
     # Determine TimeStep Data to be used
     time_step_info = TimeStepData(synchronous_mode=(not args.run_async),
                                   hz_rate=args.hz_rate,
                                   phys_substepping=args.substepping)
 
     # Apply Settings
-    apply_world_settings(client=client,
-                         world=world,
+    world = apply_world_settings(client=client,
                          time_step_info=time_step_info,
                          map_name=args.load_map,
                          force_map_reload=args.force_reload)
```

Comment:
```suggestion
                                 time_step_info=time_step_info,
                                 map_name=args.load_map,
                                 force_map_reload=args.force_reload)
```
Align indentation :smiley:

Reply:
Done